edward/inferences.py

```diff
@@ -167,12 +224,40 @@ def initialize(self, n_minibatch=1, score=None, *args, **kwargs):
         return VariationalInference.initialize(self, *args, **kwargs)
 
     def update(self):
+        """Runs one iteration of MFVI.
+
+        Returns
+        -------
+        loss : double
+            MFVI loss function value after one iteration.
+        """
         sess = get_session()
         feed_dict = self.variational.np_dict(self.zs)
         _, loss = sess.run([self.train, self.loss], feed_dict)
         return loss
 
     def build_loss(self):
+        """Wrapper for the MFVI loss function.
+
+        .. math:: 
+
+            -ELBO =  -E_{q(z; \lambda)} [ \log p(x, z) - \log q(z; \lambda) ]
+
+        MFVI supports
+
+        1. score function gradients
+        2. reparameterization gradients
+
+        of the loss function.
+
+        If the variational model is a Gaussian distribution, then part of the
+        loss function can be computed analytically.  
```

Comment:
Can we write more technically, that "If the variational model is a normal distribution and the prior is standard normal, then part of the loss function can be computed analytically following Kingma and Welling (2014), ... [insert E[log p(x|z) + KL], where the KL is analytic".